internal/api/asset_controller.go

```diff
@@ -87,7 +87,7 @@ func (ct *AssetController) GetAssetRvalue(c *gin.Context) {
 		c.Error(err)
 		return
 	}
-	c.JSON(http.StatusOK, NewDLCDataReponse(oracleInstance.PublicKey, dlcData))
+	c.JSON(http.StatusOK, NewDLCDataResponse(oracleInstance.PublicKey, dlcData))
```

Comment:
Am I crazy or there is not diff here?

Reply:
a small `Reponse` -> `Response`
while updating code I saw this, so I though may as well update :)

Reply:
Oh right thanks!